### DISCUSSION TITLE
xr.merge() of xr.DataArray's using average for overlapping "cells" of arrays

### DISCUSSION THREAD

**Discussion by mliukis** [pydata/xarray#6950] [Q&A] [answered]
Just wonder if I am missing some existing `xarray` functionality to merge multiple `xr.DataArrays` into one by taking average value of overlapping cells? 

`xr.merge()` would seem to be the perfect candidate if it is to allow averaging the values. Otherwise it seems I would need to implement averaging while merging data arrays.

Many thanks in advance!

**Comment by andersy005** [comment, marked as answer]
@mliukis, the discussion in this issue might be relevant to your use case:

- https://github.com/pydata/xarray/issues/6846

**Reply by mliukis** [reply]
@andersy005 Thank you! That does the trick - so I was missing on existing `xarray` functionality :)
